frontend/components/CourseFieldSelector.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="courses">
+    <select class="selector" v-model="mychoice" name="Kierunek">
+      <option :value="null" disabled hidden>Kierunek</option>
+      <option value="Informatyka">Informatyka</option>
+      <option value="Grafika">Grafika</option>
+      <option value="Ekonomia">Ekonomia</option>
+    </select>
+  </div>
+</template>
+
+<script>
+export default {
+  el: "#courses",
+  data() {
+    return {
+      mychoice: null,
```

Comment:
```suggestion
      myChoice: null,
```

Reply:
Ok, already chagned.

Reply:
did u push commit? i am not not see changes.